### ISSUE TITLE
UnicodeDecodeError caused by long dash character (–)

### ISSUE THREAD

**Issue by ryanjohara** [bruth/restlib2#8]
This issue was originally reported [here](https://github.com/cbmi/cilantro/issues/724) but was deemed a restlib2 issue. It was found while testing the "Email Message" field in the "Save/Share Query" feature of [cilantro](https://github.com/cbmi/cilantro). Below is the full stack trace:

```
UnicodeDecodeError at /api/queries/
'ascii' codec can't decode byte 0xe2 in position 130: ordinal not in range(128)
...
Traceback:
File "/home/devuser/webapps/pcgc-env/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  113.                         response = callback(request, *callback_args, **callback_kwargs)
File "/home/devuser/webapps/pcgc-env/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  89.         response = view_func(request, *args, **kwargs)
File "/home/devuser/webapps/pcgc-env/lib/python2.7/site-packages/restlib2/resources.py" in __call__
  269.         return self.dispatch(request, *args, **kwargs)
File "/home/devuser/webapps/pcgc-env/lib/python2.7/site-packages/restlib2/resources.py" in dispatch
  274.         response = self.process_request(request, *args, **kwargs)
File "/home/devuser/webapps/pcgc-env/lib/python2.7/site-packages/restlib2/resources.py" in process_request
  786.                     data = serializers.decode(content_type, request.body.decode())

Exception Type: UnicodeDecodeError at /api/queries/
Exception Value: 'ascii' codec can't decode byte 0xe2 in position 130: ordinal not in range(128)
```

**Comment by murphyke** [comment]
I could try my hand at this, although most of the effort will be in pep8 compliance.

**Comment by bruth** [comment]
Fixed in #9
